components/clock.rst

```diff
@@ -60,10 +62,13 @@ The Clock component also provides the ``now()`` function::
 
     use function Symfony\Component\Clock\now;
 
-    // Get the current time as a DateTimeImmutable instance
+    // Get the current time as a DatePoint instance
```

Comment:
```suggestion
    // Get the current time as a :class:`Symfony\\Component\\Clock\\DatePoint` instance
```

Reply:
This is a code example, we cannot use rst syntax here.